helpers/test.py

```diff
@@ -51,7 +51,7 @@ def run_tests(self):
             import pytest
             if not self.pytest_args:
                 self.pytest_args = ['sherpa',]
-            errno = pytest.main(self.pytest_args)
+            errno = pytest.main([self.pytest_args,])
```

Comment:
I see the tests for the previous stuff have failed anyway; looks like missing the coverage module?

Isn't automated testing fun?